guides/source/getting_started.md

```diff
@@ -1649,14 +1649,14 @@ $ bin/rails generate controller Comments
 
 This creates five files and one empty directory:
 
-| File/Directory                               | Purpose                                  |
-| -------------------------------------------- | ---------------------------------------- |
-| app/controllers/comments_controller.rb       | The Comments controller                  |
-| app/views/comments/                          | Views of the controller are stored here  |
-| test/controllers/comments_controller_test.rb | The test for the controller              |
-| app/helpers/comments_helper.rb               | A view helper file                       |
-| app/assets/javascripts/comment.coffee        | CoffeeScript for the controller          |
-| app/assets/stylesheets/comment.scss          | Cascading style sheet for the controller |
+| File/Directory                                | Purpose                                  |
+| --------------------------------------------  | ---------------------------------------- |
+| app/controllers/comments_controller.rb        | The Comments controller                  |
+| app/views/comments/                           | Views of the controller are stored here  |
+| test/controllers/comments_controller_test.rb  | The test for the controller              |
+| app/helpers/comments_helper.rb                | A view helper file                       |
+| app/assets/javascripts/comments.coffee        | CoffeeScript for the controller          |
+| app/assets/stylesheets/comments.scss          | Cascading style sheet for the controller |
```

Comment:
Still not correct in indentation. Delete the extra spaces you added and then remove one space each from the bottom two rows that you added a character to ;)